api/src/main/java/org/pmiops/workbench/db/dao/RDRExportDao.java

```diff
@@ -0,0 +1,27 @@
+package org.pmiops.workbench.db.dao;
+
+import java.math.BigInteger;
+import java.util.List;
+import org.pmiops.workbench.db.model.DbRDRExport;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.CrudRepository;
+
+public interface RDRExportDao extends CrudRepository<DbRDRExport, Long> {
+
+  @Query(
+      nativeQuery = true,
+      value =
+          "select u.user_id from user u LEFT JOIN rdr_export rdr on"
+              + " u.user_id = rdr.export_id and rdr.entity = 1 where u.last_modified_time > rdr.last_Export_date or rdr.export_id is null")
+  List<BigInteger> findDbUserToExport();
```

Comment:
This gives an error if i use Long as the return type, i suspect it is because i am using native query rather than mapping it to DBUser. i'll add comments around it